src/types/Config.tsx

```diff
@@ -181,87 +187,93 @@ export const ALL_PAGES: IPage[] = [
   },
   {
     name: PAGES.CONNECTOR,
-    role: ROLES.CONNECTOR_SETUP,
+    allowTo: () => userHasPortalRole(ROLES.CONNECTOR_SETUP),
     element: <Connector />,
   },
   {
     name: PAGES.ACCOUNT,
-    role: ROLES.MY_ACCOUNT,
+    allowTo: () => userHasPortalRole(ROLES.MY_ACCOUNT),
     element: <MyAccount />,
   },
   {
     name: PAGES.NOTIFICATIONS,
-    role: ROLES.NOTIFICATION_VIEW,
+    allowTo: () => userHasPortalRole(ROLES.NOTIFICATION_VIEW),
     element: <NotificationCenter />,
   },
   {
     name: PAGES.ORGANIZATION,
-    role: ROLES.MY_ORGANIZATION_VIEW,
+    allowTo: () => userHasPortalRole(ROLES.MY_ORGANIZATION_VIEW),
     element: <Organization />,
   },
   {
     name: PAGES.PARTNER_NETWORK,
-    role: ROLES.PARTNER_NETWORK_VIEW,
+    allowTo: () => userHasPortalRole(ROLES.PARTNER_NETWORK_VIEW),
```

Comment:
@oyo  here we need the BPDM role inside the user account.
<img width="209" alt="image" src="https://github.com/eclipse-tractusx/portal-frontend/assets/94133633/f88d44b4-5dac-42c4-be4a-4592a7ef6b94">